packages/components/modules/notifications/native/NotificationsList/index.tsx

```diff
@@ -68,8 +71,20 @@ const NotificationsList: FC<NotificationsListProps> = ({
     if (notifications.length === 0) return <EmptyState />
 
     return (
-      <View>
-        <FlatList data={notifications} renderItem={({ item }) => renderNotificationItem(item)} />
+      <View style={{ height: '100%', paddingBottom: 74 }}>
+        <InfiniteScrollerView
+          data={notifications}
+          renderItem={({ item }: { item: NotificationsListFragment$data }) =>
```

Comment:
lets change types a little bit:
```
// this can be on common
  type Notifications = NonNullable<NotificationsListFragment$data['notifications']>
  type NotificationsEdges = Notifications['edges']
  type NotificationsNode = NonNullable<NotificationsEdges[number]>['node']
  
  const renderNotificationItem = (notification: NotificationsNode) => {...}
  ...
          <InfiniteScrollerView
          data={notifications}
          renderItem={({ item }: { item: NotificationsNode }) =>
            renderNotificationItem(item)
          }
          ...
  
```